### ISSUE TITLE
Leaderboard

### ISSUE THREAD

**Issue by Apsain1** [stackup-dev/open-source-campaign#213]
**Name of Proposer**
Apoorv

**Is your feature request related to a problem? Please describe.**
The leaderboard will make the site more interactive and inclusive so the fellow stackies don't feel lonely in this big world.

**Describe the solution you'd like**
A leaderboard which shows all the stackies according to their experience on StackUp.

**Describe alternatives you've considered**
A gamified space where we can interact with fellow stackies and can compete with each other.

**Additional context**
I think this will make the whole StackUp community bond stronger.

**Comment by sayan1172** [comment]
yes i agree with your proposal and i want to add on like after every campaign there should be specific leaderboard for that campaign only and this will make more fun interactive for stackies. thanks

**Comment by Ayushgh1017** [comment]
I am also agreed with the points above. The leaderboard should be there to see our rank in the stackup community. It will encourage us to do more in the next campaign.  There is one more suggestion the stackup should introduce the chatrooms or community discussion to make this more interactive with fellow members.

**Comment by Thav33** [comment]
Awesome one. A leaderboard feature for every campaign and an overall leaderboard would be a good implementation. Also EXP rewards can be given to the top 3, 5 or 10... etc.
